crates/sui-config/src/genesis_config.rs

```diff
@@ -105,6 +105,7 @@ pub struct ValidatorGenesisInfo {
     pub commission_rate: u64,
     pub narwhal_primary_address: Multiaddr,
     pub narwhal_worker_address: Multiaddr,
+    pub narwhal_internal_worker_address: Option<Multiaddr>,
```

Comment:
I know i'm late to the party here, but this type of information should not be apart of genesis as it is private information for a node operator and not needed to be shared with anyone else.